shared/js/media/jpeg_metadata_parser.js

```diff
@@ -352,8 +352,8 @@ function parseJPEGMetadata(file, metadataCallback, metadataError) {
     case 4: // LONG
       return data.getUint32(offset, byteorder);
     case 5: // RATIONAL
-      return data.getUint32(offset, byteorder) /
-        data.getUint32(offset + 4, byteorder);
+      return [ data.getUint32(offset, byteorder),
+               data.getUint32(offset + 4, byteorder) ];
```

Comment:
actually I think this change should never have made it to the patch as I'm not using that reader anyway. it was a left over from the previous version....